README.md

```diff
@@ -121,6 +121,18 @@ Note that there is a currently a race condition that means that the client test
 some server test suite requests. As a work-around, run the server test suite a second time while the
 client is waiting for clinical data requests during test 1.2.
 
+## Running the Bulk Export Server Tests
```

Comment:
Another demonstration that would be valuable would be to show how to run against the PDex Client tests. I tried this and it didn't get very far because the bulk tests stop the workflow if they detect even small things like bad HTTP statuses (see [FI-3949](https://oncprojectracking.healthit.gov/support/browse/FI-3949)). Probably doesn't make sense to resolve that or fixes to make this test kit return the right statuses as a part of this ticket, but wanted to mention it.